### ISSUE TITLE
Login does not work 

### ISSUE THREAD

**Issue by hemangjoshi37a** [getcursor/cursor#1323]
1) If you can, please include a screenshot of your problem
![Screenshot from 2024-03-17 12-08-18](https://github.com/getcursor/cursor/assets/12392345/6e65d249-8587-41e7-b9f0-5cad2fec52c1)

2) Please include the name of your operating system
![Screenshot from 2024-03-17 12-08-47](https://github.com/getcursor/cursor/assets/12392345/e0e3d50b-d86c-4304-bbca-7091e9c8d043)

3) If you can, steps to reproduce are super helpful
Click on login button and nothing happens, login button is not working.

**Comment by hemangjoshi37a** [comment]
I don't know why it is not getting solved even though there are so many people facing similar issue for long time. also can't find any source code so that any community member can fix this . It works great with windows 11 but not with ubuntu .

**Comment by NarcissusNull** [comment]
> I has the same issue in my arch linux. so, Can I get any help from cursor teams? I have paid for cursor pro!

Sorry for what I said. I fix this issue by using proxy in vscode settings.

**Comment by hemangjoshi37a** [comment]
how to use this proxy exactly sir ?

**Comment by hemangjoshi37a** [comment]
@TheOddler at this point it is hopeless. It is better to use other alternatives

**Comment by tringler** [comment]
same here on `Ubuntu 24.04.2 LTS`

**Comment by osternaudClem** [comment]
Just get this issue right now. I use firefox as default browser.
I installed Chrome, and the sign in page opened with that browser.

**Comment by TheOddler** [comment]
>  I installed Chrome, and the sign in page opened with that browser.

This worked, though I also had to mark Chrome as the default browser. Also Chromium (which is available in the App Center) doesn't work, had to be Chrome (which isn't in the App Center, so have to manually download it).

**Comment by garipovazamat** [comment]
The same problem. OS: Ubuntu 22.04.2 LTS. In syslog i see cursor trying to open my default browser. My default browser is Opera.

**Comment by GhouI** [comment]
> > I installed Chrome, and the sign in page opened with that browser.
> 
> This worked, though I also had to mark Chrome as the default browser. Also Chromium (which is available in the App Center) doesn't work, had to be Chrome (which isn't in the App Center, so have to manually download it).

Yeah I did the same as this and it worked. Unfortunately it seems it doesn't work with other browsers besides Chromium.

**Comment by spiritix** [comment]
I have Chrome installed and it still doesn't work for me. Will switch to Windsurf.

**Comment by AdnanTemur** [comment]
This worked for me. Download the app image (This case Cursor-0.48.8-x86_64.AppImage).
`./Cursor-0.48.8-x86_64.AppImage --no-sandbox`

or

`Cursor-0.48.8-x86_64.AppImage --no-sandbox --appimage-extract`

**Comment by aruprakshit** [comment]
I am using Ubuntu from wsl2. I had the same problem. I right click on the sign in link, and then pasted the link in the browser(chrome), and then the login worked as expected. But clicking the sign in button from cursor window was not working, but manually copying the link worked.

```
h2so4@deadpool:~/kafka$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 24.04.3 LTS
Release:        24.04
Codename:       noble
h2so4@deadpool:~/kafka$
```
